app/mailers/application_mailer.rb

```diff
@@ -2,6 +2,6 @@
 
 class ApplicationMailer < ActionMailer::Base
   helper :application # gives access to all helpers defined within `application_helper`.
-  default from: 'from@example.com'
+  default from: 'stephanie.funk@midatlanticbabywearing.org'
```

Comment:
I think I would rather see this in config/environment and the value tied to an environment variable.